shared/teams/team/rows/channel-row/channel.tsx

```diff
@@ -24,6 +25,11 @@ const ChannelRow = (props: ChannelRowProps) => {
   const canPerform = Container.useSelector(state => Constants.getCanPerformByID(state, teamID))
   const canDelete = canPerform.deleteChannel
 
+  const numParticipants = Container.useSelector(
+    state => ChatConstants.getParticipantInfo(state, conversationIDKey).all.length
```

Comment:
we can compare this to the number of members in `teamDetails` for `hasAllMembers`